lib/active_encode/engine_adapters/media_convert_adapter.rb

```diff
@@ -205,9 +205,25 @@ def convert_output(job)
              :audio_bitrate, :video_bitrate, :file_size, :label, :url, :id].each do |field|
               output.send("#{field}=", tech_md[field])
             end
-            output.id ||= "#{job.id}-output-#{index}"
+            output.id ||= "#{job.id}-output#{tech_md[:suffix]}"
```

Comment:
Does the suffix start with a `-` or does one need to be added?

Reply:
Suffix usually starts with something to set it apart from the base. The way I see it, is the modifier is ok to append to the filename without an additional separator (which is how MediaConvert does it), it’s ok for the ID as well.